gammapy/utils/testing.py

```diff
@@ -1,20 +1,25 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 """Utilities for testing"""
 from __future__ import absolute_import, division, print_function, unicode_literals
+
 import os
+
 from astropy.tests.helper import pytest
-from astropy.utils.data import get_pkg_data_filename
+from astropy.utils import minversion
 from ..data import DataManager
 
 __all__ = [
     'requires_dependency',
     'requires_data',
+    'SHERPA_LT_4_8'
 ]
 
+# Todo : Change to 4_8 once there is a sherpa release
```

Comment:
@joleroi - I think this TODO can be resolved now!?

The Sherpa version we support and test at the moment is 4.8:
https://travis-ci.org/gammapy/gammapy/jobs/106282386#L1026
```
$ python -c 'import sherpa; print(sherpa.__version__)'
4.8.0
```
If you use the git master version it will be 4.8.something:
```
$ python2.7 -c 'import sherpa; print(sherpa.__version__)'
4.8.0+82.gbfcc195.dirty
```
Testing for `'4.7+642'` seems unnecessary / brittle to me.